### ISSUE TITLE
Undefined index: lloc in /usr/local/src/phploc/src/Analyser.php on line 306

### ISSUE THREAD

**Issue by sebastianbergmann** [sebastianbergmann/phploc#138]
```
$ wget https://raw.githubusercontent.com/moodle/moodle/v3.0.2/lib/moodlelib.php
```

```
$ phploc moodlelib.php
phploc 3.0.0 by Sebastian Bergmann.

PHP Notice:  Undefined index: lloc in /usr/local/src/phploc/src/Analyser.php on line 306
PHP Stack trace:
PHP   1. {main}() /usr/local/src/phploc/phploc:0
PHP   2. Symfony\Component\Console\Application->run() /usr/local/src/phploc/phploc:31
PHP   3. SebastianBergmann\PHPLOC\CLI\Application->doRun() /usr/local/src/phploc/vendor/symfony/console/Application.php:117
PHP   4. Symfony\Component\Console\Application->doRun() /usr/local/src/phploc/src/CLI/Application.php:98
PHP   5. Symfony\Component\Console\Application->doRunCommand() /usr/local/src/phploc/vendor/symfony/console/Application.php:186
PHP   6. Symfony\Component\Console\Command\Command->run() /usr/local/src/phploc/vendor/symfony/console/Application.php:787
PHP   7. SebastianBergmann\PHPLOC\CLI\Command->execute() /usr/local/src/phploc/vendor/symfony/console/Command/Command.php:256
PHP   8. SebastianBergmann\PHPLOC\CLI\Command->executeSingle() /usr/local/src/phploc/src/CLI/Command.php:109
PHP   9. SebastianBergmann\PHPLOC\CLI\Command->count() /usr/local/src/phploc/src/CLI/Command.php:128
PHP  10. SebastianBergmann\PHPLOC\Analyser->countFiles() /usr/local/src/phploc/src/CLI/Command.php:236
PHP  11. SebastianBergmann\PHPLOC\Analyser->countFile() /usr/local/src/phploc/src/Analyser.php:161
```

**Comment by calbrecht** [comment]
If you put an anonymous function with use statement inside the first global function in source.php, you will also get this notice if E_ALL.

    function &a_global_function()
    {
        $a = AClass::CLASS;
        $f = function () use ($a) { return ''; };
    }

**Comment by sebastianbergmann** [comment]
Should be taken care of by 2fa2e75fcb23dca6f277f5bbc12c2669c353a447.

**Comment by hackel** [comment]
I also started seeing this issue recently.  I tried updating to 3.0.1, but am still getting it:
```
PHP Notice:  Undefined index: ccn in /home/quirks/vendor/phploc/phploc/src/Analyser.php on line 317
PHP Notice:  Undefined index: lloc in /home/quirks/vendor/phploc/phploc/src/Analyser.php on line 306
```
In my case (after much trial and error to narrow it down), I figured out that it was caused by a method I had named `list`, which is a reserved keyword.  Simply renaming this method caused the notices to disappear.  

I realise phploc isn't a linter, but it would still be helpful to get a more useful error message.  (Incidentally, my code passed `php -l` with no syntax errors detected.)
